modules/apps/friendly-url/friendly-url-web/src/main/java/com/liferay/friendly/url/web/internal/portlet/action/FriendlyURLSeparatorSaveCompanyConfigurationMVCActionCommand.java

```diff
@@ -121,6 +122,12 @@ private String _getFriendlyURLSeparators(
 					String friendlyURLSeparator = ParamUtil.getString(
 						actionRequest, friendlyURLResolver.getKey());
 
+					if (!friendlyURLResolver.isVisible()) {
+						friendlyURLSeparator = StringUtil.removeSubstring(
+							friendlyURLResolver.getURLSeparator(),
+							StringPool.SLASH);
+					}
```

Comment:
Could you explain what this code block is doing? Is there a way to put this logic in the Object specific resolver rather than needing to check it here for all resolvers?

Will this block still be needed after the feature flag is removed or is this only temporary?

Reply:
@jonathanmccann I removed this code as it was no longer necessary